packages/shared/src/mock/values.ts

```diff
@@ -1,189 +1,241 @@
 import { range } from 'lodash';
+import type { NdArray } from 'ndarray';
+import ndarray from 'ndarray';
 
-const arr1 = range(-20, 21);
-const arr2 = range(0, 100, 5);
-const arr3 = range(-1, 1.25, 0.25);
-const arr4 = range(10, 40, 10);
-const arr5 = range(1, 4);
-const arr6 = range(9335, 9338);
+import type { ArrayValue, DType, H5WebComplex } from '../models-hdf5';
 
-const oneD = arr1.map((val) => val ** 2);
-const twoD = arr2.map((offset) => oneD.map((val) => val - offset));
-const threeD = arr3.map((multiplier) =>
-  twoD.map((arrOneD) => arrOneD.map((val) => val * multiplier)),
-);
-const fourD = arr4.map((divider) =>
-  threeD.map((arrTwoD) =>
-    arrTwoD.map((arrOneD) => arrOneD.map((val) => Math.sin(val / divider))),
-  ),
-);
+export function cplx(real: number, imag: number): H5WebComplex {
+  return [real, imag];
+}
```

Comment:
Small utility to generate properly typed `H5WebComplex` arrays - e.g. `cplx(1, 2)` instead of `[1, 2] as H5WebComplex`